### ISSUE TITLE
api.HTMLSelectElement - no page for 'autocomplete'

### ISSUE THREAD

**Issue by XX-J** [mdn/browser-compat-data#8799]
<details><summary>Screenshots:</summary>

![1](https://jpegshare.net/images/a0/34/a0349a202038bb6fc33b08d32f3cce04.png)
----------
![2](https://jpegshare.net/images/a0/73/a073a8150829205ea446aa0dceba5635.png)
</details>

<details>
<summary>MDN page report details</summary>

* Query: `api.HTMLSelectElement`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement
* Report started: 2021-01-18T17:09:42.254Z

</details>

**Comment by vinyldarkscratch** [comment]
Thanks for your report -- the page should be linking to https://developer.mozilla.org/docs/Web/HTML/Attributes/autocomplete instead, we'll get that resolved!

**Comment by ddbeck** [comment]
>  the page should be linking to https://developer.mozilla.org/docs/Web/HTML/Attributes/autocomplete instead, we'll get that resolved!

I don't think this is quite right. We don't usually link from properties to their reflected attributes. This should probably be like its sibling features and link to the documentation for the property itself. Unfortunately, `HTMLSelectElement.autocomplete` appears to be undocumented on MDN: it's not mentioned in the body of the `HTMLSelectElement` page and the `autocomplete` page does not yet exist.

I'm going to edit the description for this issue and transfer this issue to [mdn/content](https://github.com/mdn/content/).

Thanks for reporting this, @XX-J!
